pkg/config/config.go

```diff
@@ -42,12 +43,50 @@ type Common struct {
 }
 
 type Redis struct {
-	Address  string `yaml:"address"`
-	Password string `yaml:"password"`
-	MetaDB   int    `yaml:"metaDB"`
-	QueueDB  int    `yaml:"queueDB"`
-	LockDB   int    `yaml:"lockDB"`
-	ConfigDB int    `yaml:"configDB"`
+	Address  string        `yaml:"address"`
+	Sentinel RedisSentinel `yaml:"sentinel"`
+	Cluster  RedisCluster  `yaml:"cluster"`
```

Comment:
This is more clear approach but it will break backwards compatibility. If someone will update chorus version it will not start untill config will be updated. So my idea was just add new parameters and don't modify existing ones

Reply:
If the backward compatibility is the only thing that is desired, then it can be achieved just by leaving `address` key alongside with `addresses`. It'd be simpler to write and to parse, and will also leave a space to deprecate `address` field in the future if needed.

Reply:
agreed, done